lib/decorateWithMetrics.js

```diff
@@ -24,8 +24,6 @@ function decorateWithMetrics(func, alias = func.name) {
         throw new Error(`"${alias}" is already decorated`);
     }
 
-    stats[alias] = [];
-
```

Comment:
We don't need this here, it's created during the `finally` call

Reply:
Reverting this change as it's part of a test and functionality to prevent duplicate decorations
I don't think we need this functionality but it will be perhaps to much changes to remove it under this PR